packages/teams-js/src/public/tasks.ts

```diff
@@ -66,7 +65,7 @@ export namespace tasks {
 
   /**
    * @deprecated
-   * As of 2.0.0, please use {@link dialog.submit dialog.submit(result?: string | object, appIds?: string | string[]): void} instead.
+   * As of 2.0.0, please use {@link dialog.submit} instead.
```

Comment:
@erikadoyle I was not able to get this @link tag to resolve (and by "resolve" I mean stop producing a warning when I run "typedoc --emit none --validation.invalidLink src/index.ts") without this change. The function signature was copied exactly from the dialog.submit function so I can't figure out why this isn't working.

The [public documentation](https://docs.microsoft.com/en-us/javascript/api/@microsoft/teams-js/tasks?view=msteams-client-js-latest#@microsoft-teams-js-tasks-submittask) doesn't resolve to dialog.submit even though the link doesn't appear obviously broken.
Do you know how to get this to work?

Reply:
Hmm, looks like none of the @deprecated note links in [tasks module](https://docs.microsoft.com/en-us/javascript/api/@microsoft/teams-js/tasks?view=msteams-client-js-latest#@microsoft-teams-js-tasks-submittask) are working. I took a look but couldn't discern why `{@link dialog.submit} isn't resolving to the intended target:

https://docs.microsoft.com/en-us/javascript/api/@microsoft/teams-js/dialog?view=msteams-client-js-latest#@microsoft-teams-js-dialog-submit

. . . and instead reloading the current page:

https://docs.microsoft.com/en-us/javascript/api/@microsoft/teams-js/tasks?view=msteams-client-js-latest

FWICT this is the correct syntax per [Typedoc link resolution reference](https://typedoc.org/guides/link-resolution/). I'll ask MS Docs engineering about this. Hoping this is a publishing pipeline fix; how about we move forward on that assumption, and file a new task if any link source updates are needed?

Reply:
Thanks, that sounds good to me. Just so I'm being clear, looking at the source code, it looked like what was checked in _before_ my change should have worked properly. The function signature matched the destination of the link exactly, so I would have expected it was working, but as you mentioned it is not resolving.